src/OpenTelemetry.Shared/IOfflineBlob.cs

```diff
@@ -0,0 +1,48 @@
+// <copyright file="IOfflineBlob.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenTelemetry.Shared
+{
+    /// <summary>
+    /// API to Read, Write and Delete blob.
+    /// </summary>
+    public interface IOfflineBlob
```

Comment:
Should we use "Offline" in interface names? In case of local otlp agent scenario, this feature can be used even when app is online, but agent is down. Or in general, backend is down, but app is Online (i.e no network issues)